### DISCUSSION TITLE
Few questions on ERC4337.payPrefund

### DISCUSSION THREAD

**Discussion by livingrockrises** [Vectorized/solady#921] [Q&A]
I'm going over this code https://github.com/Vectorized/solady/blob/main/src/accounts/ERC4337.sol#L172 
trying to understand / have below questions

1. Why does the result of call need to be explicity pop-ed ?
2. Why use codesize() specifically for argsOffset and returnOffset ?

@Vectorized

**Comment by Vectorized** [comment]
1. Compiler requires it.
2. 2 gas vs 2 to 3 gas.
